wdl-parsing/WDLParsingFunction/checkstyle.xml

```diff
@@ -0,0 +1,339 @@
+<?xml version="1.0"?>
```

Comment:
Try using symbolic link to avoid copy-paste
example https://github.com/dockstore/dockstore/blob/feature/languageParsingLambda/dockstore-webservice/findbugs-exclude.xml

Reply:
it's not the same file

Reply:
![Screenshot from 2020-08-14 15-28-08](https://user-images.githubusercontent.com/1730679/90286119-32856700-de43-11ea-88d9-d542ba5e7945.png)

I'm actually open to switching the coding style for the project as whole. 
But it should be done as a formatting commit. I think it's important that we collaborate with a shared coding style across our projects.